RadixWallet/Features/CreateAccount/Children/CreationOfAccount/CreationOfAccount+Reducer.swift

```diff
@@ -1,160 +0,0 @@
-import ComposableArchitecture
-import SwiftUI
-
-public struct CreationOfAccount: Sendable, FeatureReducer {
-	public struct State: Sendable, Hashable {
-		public let name: NonEmptyString
-		public let networkID: NetworkID?
-		public let isCreatingLedgerAccount: Bool
-		public enum Step: Sendable, Hashable {
-			case step0_chooseLedger(LedgerHardwareDevices.State)
-			case step1_derivePublicKeys(DerivePublicKeys.State)
-		}
```

Comment:
This reducer is deleted since:
- the choose ledger flow can be simply triggered from the coordinator using the `LedgerHardwareDevices` as a step
- the derive public keys flow is no wanted as a child of `CreateAccount` that gets pushed